overlord/configstate/configcore/ctrlaltdel.go

```diff
@@ -0,0 +1,142 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package configcore
+
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/logger"
+	"github.com/snapcore/snapd/overlord/configstate/config"
+	"github.com/snapcore/snapd/sysconfig"
+	"github.com/snapcore/snapd/systemd"
+)
+
+// Systemd unit name to control ctrl-alt-del behaviour
+const (
+	ctrlAltDelTarget = "ctrl-alt-del.target"
+)
+
+// Supported actions for core.system.ctrl-alt-del-action
+const (
+	ctrlAltDelReboot = "reboot"
+	ctrlAltDelNone   = "none"
+)
+
+func init() {
+	supportedConfigurations["core.system.ctrl-alt-del-action"] = true
+}
+
+type sysdCtrlAltDelLogger struct{}
+
+func (l *sysdCtrlAltDelLogger) Notify(status string) {
+	logger.Noticef("CtrlAltDel handler: %s", status)
```

Comment:
historically our other loggers around here seems to print to Stderr, maybe @mvo5 remembers why?

Reply:
Actually the only place it's called at is:
https://github.com/snapcore/snapd/blob/99130902dd740b66fe47248ec5745eda719f3b3c/systemd/systemd.go#L957-L958

so our code that prints `<some random prefix>: <msg>` isn't perhaps doing the right thing, eg:
https://github.com/snapcore/snapd/blob/99130902dd740b66fe47248ec5745eda719f3b3c/overlord/configstate/configcore/backlight.go#L42-L44

perhaps systemd package should just take nil for reporter and we can then drop that unnecessary code

Reply:
I switched it here to also use Stderr for consistency and we can have a followup that makes it a logger.